src/applications/disability-benefits/781/pages/informationInterviewAssault.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { IncidentIntroduction781 } from '../helpers';
+
+export const uiSchema = {
+  'ui:description': ({ formData }) => (
+    <IncidentIntroduction781 formData={formData} formType="781a" />
```

Comment:
Unless there is a good reason to, we should try and avoid naming the form in any of the reusable function names.

If this isn't reusable, then the form name is not correct.